lib/ts/recipe/dashboard/api/userroles/roles/createRole.ts

```diff
@@ -0,0 +1,47 @@
+import { APIInterface, APIOptions } from "../../../types";
+import UserRoles from "../../../../userroles";
+
+import STError from "../../../../../error";
+
```

Comment:
this API should not be needed, since there is already another API that calls createNewRoleOrAddPermissions